Tasks/ServiceFabricDeploy/task.json

```diff
@@ -16,8 +16,11 @@
     "version": {
         "Major": 1,
         "Minor": 3,
-        "Patch": 5
+        "Patch": 6
     },
+    "demands": [
+        "Cmd"
```

Comment:
Isn't the dependency on powershell rather than on windows, and hence would it be better to use "Powershell" demand here.